server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -284,7 +284,7 @@ export function configureWebSocketServices(
                 }
                 // We don't understand the error, so it is likely an internal service error.
                 const errMsg = `Could not verify connect document token. Error: ${safeStringify(error, undefined, 2)}`;
-                return handleServerError(logger, errMsg, claims.tenantId, claims.documentId);
+                return handleServerError(logger, errMsg, claims.documentId, claims.tenantId);
```

Comment:
@znewton - FYI, found wrong order of arguments by glancing where notion of "documentId" is used. Please glance at my comments for interfaces, if they need to be changed to better reflect FRS reality.

Reply:
Thanks for this change!